src/tactic/interactive_expr.lean

```diff
@@ -153,6 +161,12 @@ meta def view {γ} (tooltip_component : tc subexpr (action γ)) (click_address :
     on_click (λ _, action.on_click ca),
     key s
   ] [html.of_string s]]
+| ca (sf.block i a) := do
+  inner ← view ca a,
+  pure [h "span" [cn "indent-code", style [("--indent-level", to_string i)]] inner]
```

Comment:
Did you try this out in vscode (i.e., it doesn't crash)?  Last time I ran into few issues (but maybe that was because I tried to use `attr.val "style" "..."`).

Reply:
Yeah you aren't allowed to do `attr.val "style"` :/

Reply:
I was developing all this in the lean solution, but yes, these tags worked fine. Basically they do nothing unless you have a CSS class defined.